node/src/gql/mutation/login-google.ts

```diff
@@ -51,7 +53,8 @@ export const loginGoogle = {
   },
   resolve: async (
     _root: GraphQLObjectType,
-    args: { accessToken: string }
+    args: { accessToken: string },
+    context: any
```

Comment:
causes type errors if we use a specific type for `context` here? I know the graphql types can be tricky. Only use `any` in places you absolutely must and then, you will need to `// eslint-disable-next-line <whatever is the rule about no any>`